src/System.Private.CoreLib/shared/System/IO/Stream.cs

```diff
@@ -189,6 +189,74 @@ private int GetCopyBufferSize()
             return bufferSize;
         }
 
+        public virtual void CopyTo(ReadOnlySpanAction<byte, object?> callback, object? state, int bufferSize)
+        {
+            StreamHelpers.ValidateCopyToArgs(this, callback, bufferSize);
```

Comment:
Why do we need to do all of this validation?  The CopyTo method we're delegating to is going to do almost all of it for us.  Don't we need to just validate that callback isn't null?

Reply:
Fixed.